app/services/katello/pulp3/pulp_content_unit.rb

```diff
@@ -116,24 +116,30 @@ def self.fetch_content_list(page_opts)
         content_unit_list page_opts
       end
 
+      # rubocop:disable Lint/UselessAssignment
       def self.find_duplicate_unit(repository, unit_type_id, file, checksum)
+        filter_label = 'sha256'
+        if unit_type_id == 'ostree_ref'
```

Comment:
What if `filter_label` was one if the fields for the content type? Then we wouldn't need a conditional. And maybe it could be 'sha256' by default since that's what everything else uses.

Reply:
actually i meant to comment on this.  Given the current changes, i don't think this is needed.  we never actually want to re-use a content unit within the upload process, only an artifact (which we can still lookup via sha256).  So i don't think this code path will ever actually be hit.  thoughts @jjeffers ?